website/docs/d/customer_subnets.html.markdown

```diff
@@ -25,14 +25,16 @@ data "vmc_customer_subnets" "my_subnets" {
 
 * `num_hosts` - (Optional) The number of hosts.
 
-* `connected_account_id` - (Optional) The linked connected account identifier.
+* `connected_account_id` - (Required) The linked connected account identifier.
 
 * `sddc_id` - (Optional) SDDC identifier.
 
-* `force_refresh` - (Optional) Boolean value when set to true, forces the mappings for datacenters to be refreshed for the connected account.
+* `force_refresh` - (Optional) When true, forces the mappings for datacenters to be refreshed for the connected account.
 
 * `instance_type` - (Optional) The server instance type to be used.
 
+* `sddc_type` - (Optional) The sddc type to be used. (1NODE, SingleAZ, MultiAZ)
```

Comment:
For all the optional params, can we specify what the default would be if its not explicitly set

Reply:
This is a  GET API. Optional parameters are usually used as a filter for the query. So if they are not provided by the user, the GET returns all the results (all customer subnets in our case). But if optional parameters are provided by the user they will act as a filter criteria.

Reply:
Got it. Thanks